css/styles.css

```diff
@@ -1,51 +1,209 @@
-* {
-  font-family: Lato, sans-serif;
+.container {
+  font-family: "Proza Libre", sans-serif;
+}
+
+body {
+  background: #f0f0f0;
 }
 
 header {
-  background-color: #4b2373;
-  color: white;
-  padding: 15px;
-  font-family:
-    "Proza Libre",
-    sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 27px;
+  color: #fff;
+  background-color: #562b81;
+}
+
+.new-entry {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 30px;
+  line-height: 41px;
+  padding-top: 16px;
+  padding-bottom: 9px;
 }
 
 img {
+  border-radius: 4px;
+  max-width: 100%;
+}
+
+.minor-heading {
+  font-style: normal;
+  font-weight: 600;
+  font-size: 16px;
+  line-height: 22px;
+  padding-bottom: 9px;
+  padding-top: 25px;
+  margin-top: 5px;
+}
+
+.small-box {
+  width: 100%;
+  height: 40px;
+  margin-bottom: 10px;
+}
+
+.large-box {
+  width: 100%;
+  height: 149px;
+}
+
+.inputs {
   width: 100%;
 }
 
-.form-actions {
+.button {
   display: flex;
   justify-content: flex-end;
 }
 
-label {
-  display: block;
+button {
+  font-family: "Proza Libre", sans-serif;
+  font-size: 16px;
+  color: #f0f0f0;
+  background: #562b81;
+  border-radius: 4px;
+  position: absolute;
+  width: 72px;
+  height: 32px;
+  left: none;
+  top: none;
 }
 
-input,
-textarea {
-  width: 100%;
-  margin-bottom: 10px;
+input:focus,
+textarea:focus {
+  outline: none;
+  border: 2px solid #562b81;
+  box-shadow: 0 0 1px 1px (#562b81/ 60%);
 }
 
-textarea {
-  height: 130px;
+#id {
+  position: inherit;
 }
 
-.row {
-  font-weight: bold;
+.is-flex {
+  justify-content: space-between;
 }
 
-input,
-textarea:focus {
-  outline-color: #4b2373;
-  width: 100%;
+.entries-desktop {
+  margin: 0;
+  justify-content: space-between;
 }
 
-button {
-  cursor: pointer;
-  color: white;
-  background-color: #4b2373;
+.padding {
+  padding: 20px;
+}
+
+@media screen and (min-width: 768px) {
+  .entry-form {
+    width: 100%;
+    margin-left: auto;
+    margin-right: auto;
+  }
+
+  .display {
+    display: flex;
+  }
+
+  .inputs {
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-start;
+  }
+
+  .minor-heading {
+    padding-top: 0;
+  }
+
+  .image {
+    width: 100%;
+    margin-right: 10px;
+  }
+
+  .button {
+    display: flex;
+    justify-content: flex-end;
+  }
+
+  .button button {
+    position: unset;
+  }
+
+  .entries {
+    width: 100%;
+  }
+
+  .entries-header {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .entry-display {
+    width: 100%;
+    display: flex;
+  }
+
+  .image-desktop {
+    float: left;
+  }
```

Comment:
I don't think this is being used anywhere? In either case, you should avoid using `float`